release_images_inference.yml

```diff
@@ -43,3 +43,53 @@ release_images:
       # to images being published.
       force_release: False   # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
+  4:
+    framework: "pytorch"
+    version: "2.0.0"
+    arch_type: "x86"
+    customer: "ec2"
+    inference:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py310" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu118"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  5:
+    framework: "pytorch"
+    version: "2.0.0"
+    arch_type: "x86"
+    customer: "sagemaker"
```

Comment:
param is `customer_type` - these will need to be changed throughout

Reply:
Thank you for the catch. it's updated to be customer_type on all 4 instances.